### PR TITLE
Installing confluent-telemetry package as a part of docker build.

### PR DESCRIPTION
**Pull request by harinirajendran**

### CODE REVIEW

File: cp-ksqldb-cli/Dockerfile.ubi8 [hunk 1]
```diff
@@ -53,7 +53,12 @@ gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
 enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+    && yum install -y \
+        confluent-hub-client-${CONFLUENT_VERSION} \
+        # We are installing confluent-telemetry package explicitly because
+        # KSQL's deb/rpm packages cannot directly depend on
+        # confluent-telemetry package as KSQL is Open Source.
+        confluent-telemetry-${CONFLUENT_VERSION} \
```
Comment (AlanConfluent) [inline cp-ksqldb-cli/Dockerfile.ubi8:61]: Maybe keep with the existing pattern and echo what you're installing before installing it

File: cp-ksqldb-server/Dockerfile.ubi8 [hunk 2]
```diff
@@ -58,7 +58,12 @@ enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && echo "===> Installing Confluent security plugins ..." \
     && yum install -y confluent-security-${CONFLUENT_VERSION} \
     && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+    && yum install -y \
+        confluent-hub-client-${CONFLUENT_VERSION} \
+        # We are installing confluent-telemetry package explicitly because
+        # KSQL's deb/rpm packages cannot directly depend on
+        # confluent-telemetry package as KSQL is Open Source.
```
Comment (AlanConfluent) [inline cp-ksqldb-server/Dockerfile.ubi8:65]: Same here

File: cp-ksqldb-cli/Dockerfile.ubi8 [hunk 3]
```diff
@@ -53,7 +53,12 @@ gpgcheck=1 \n\
 gpgkey=${CONFLUENT_PACKAGES_REPO}/archive.key \n\
 enabled=1 " > /etc/yum.repos.d/confluent.repo \
     && echo "===> Installing Confluent Hub client ..." \
-    && yum install -y confluent-hub-client-${CONFLUENT_VERSION} \
+    && yum install -y \
```
Comment (brianbushree) [inline cp-ksqldb-cli/Dockerfile.ubi8:56]: I am not sure we need this for the CLI... 🤔

Comment (brianbushree) [reply cp-ksqldb-cli/Dockerfile.ubi8]: yea, let's not install this for the CLI (we also don't install `confluent-security`)

**Pull request merged**